src/modules/comment.js

```diff
@@ -0,0 +1,47 @@
+/*eslint-disable*/
+import CountComments from './CountComments';
+
+class Comments {
+  FetchComments = (itemId) => {
+    fetch(
+      `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/tJRup0Me8cP9OobHqxUY/comments?item_id=${itemId}`,
+    )
+      .then((res) => res.json())
+      .then((data) => {
+        if (data.error) return;
+        const commentContainer = document.querySelector('.all-comments');
+        commentContainer.innerHTML = data
+          .map(
+            (element) => `<li class="comments-element">
+            ${element.creation_date.replace('-', '/')}&nbsp;${
+              element.username
+            }&nbsp:&nbsp;${element.comment}
+            </li>`,
+          )
+          .join('');
+        return CountComments(data);
+      });
+  };
+
+  PostComment = ({ commentId, name, textArea }) => {
+    console.log(commentId, name, textArea);
+    const config = {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json; charset="utf-8"',
+      },
+      body: JSON.stringify({
+        item_id: commentId,
+        username: name,
+        comment: textArea,
```

Comment:
- [ ] Please kindly ensure that a user is able to add comments as I am unable to do that at the moment.
- [ ] At the moment, when you click any part of the modal it disappears, please kindly make sure the modal only disappears when the `X` button is clicked.